frontend/packages/core/src/Table/table.tsx

```diff
@@ -132,64 +136,84 @@ interface TableProps extends Pick<MuiTableProps, "stickyHeader"> {
   children?:
     | (React.ReactElement<TableRowProps> | null | undefined | {})[]
     | React.ReactElement<TableRowProps>;
+  order?: Order;
+  orderBy?: string;
+  onRequestSort?: (event: React.MouseEvent<unknown>, property: string) => void;
 }
 
-const Table: React.FC<TableProps> = React.forwardRef(
```

Comment:
Is it intentional not to forward the ref anymore?

Reply:
nop, my bad, I'll put it back